lib/sorbet-schema/t/struct.rb

```diff
@@ -15,6 +15,8 @@ def schema
       sig { params(type: Symbol).returns(Typed::Serializer[T.untyped, T.untyped]) }
       def serializer(type)
         case type
+        when :nested_hash
```

Comment:
thought: I'm not in love with this name, but I can't really think of an alternative. `deep` comes to mind, but not sure I love that either. Any other name options you can come up with?

Reply:
Yeah, I struggled with this. I thought of `deep_hash` too. There is also `deeply_nested_hash.`

Reply:
`hash_deeply_serialized`
`hash_recursively_serialized`
`hash_with_serialized_values`
`has_fully_traversed`

Reply:
Let's go with `deeply_nested_hash` for right now. We'll probably change it in the future 😂